pages/dkp/kommander/2.2/dkp-upgrade/custom-apps/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Custom Applications
+title: Upgrade Custom Applications
+menuWeight: 60
+excerpt: Verify the compatibility of Custom Applications with the current Kubernetes version
+beta: false
+enterprise: true
+---
+
+We recommend upgrading your Custom Applications to the latest compatible version as soon as possible. Since there is no expectation of support by D2iQ for Custom Applications, you must upgrade them manually.
```

Comment:
I find the phrasing 'expectation of support' awkward.

```suggestion
We recommend upgrading your Custom Applications to the latest compatible version as soon as possible. Since Custom Applications are not created, maintained or supported by D2iQ, you must upgrade them manually.
```